terraform-tests/sqs_test.go

```diff
@@ -83,15 +85,35 @@ var _ = Describe("SQS", Label("SQS-terraform"), Ordered, func() {
 	Context("FIFO queues", func() {
 		BeforeAll(func() {
 			plan = ShowPlan(terraformProvisionDir, buildVars(defaultVars, map[string]any{
-				"fifo": true,
+				"fifo":                  true,
+				"deduplication_scope":   "queue",
+				"fifo_throughput_limit": "perQueue",
 			}))
 		})
 
 		It("should create an SQS FIFO queue with the correct properties", func() {
 			Expect(AfterValuesForType(plan, "aws_sqs_queue")).To(
 				MatchKeys(IgnoreExtras, Keys{
-					"name":       Equal(fmt.Sprintf("%s.fifo", name)),
-					"fifo_queue": BeTrue(),
+					"name":                  Equal(fmt.Sprintf("%s.fifo", name)),
+					"fifo_queue":            BeTrue(),
+					"deduplication_scope":   Equal("queue"),
+					"fifo_throughput_limit": Equal("perQueue"),
+				}),
+			)
+		})
+
+		It("should create a FIFO SQS queue for high throughput mode", func() {
```

Comment:
nitpick: I noticed this is not a new test. It looks like it was simply moved to a different Context block.
I would have tried to make the diffing reflect this. If not possible, I usually prefer creating a different PR or commit only for "reformatting".
I usually do this to distinguish between the absolute minimum changes, and the opportunistic improvements (which are always more than welcome).

Leaving that aside, thank you for taking the time to organise the tests into a more sensible structure.